frontend/benefit/applicant/src/components/attachmentsIngress/AttachmentsIngress.sc.ts

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+
+export const $TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 740px;
+  margin-bottom: ${(props) => props.theme.spacing.l};
+`;
+
+export const $Heading = styled.h1`
+  font-size: ${(props) => props.theme.fontSize.heading.m};
+  font-weight: 500;
+  margin: 0;
+`;
+
+export const $Description = styled.p`
+  font-size: ${(props) => props.theme.fontSize.heading.s};
+  font-weight: normal;
+  line-height: ${(props) => props.theme.lineHeight.l};
+`;
+
+export const $WarningContainer = styled.div`
+  display: flex;
+  align-items: center;
+  flex: 1 0 30%;
+  box-sizing: border-box;
```

Comment:
Redefining box-sizing for elements separately, as opposite to define it globally for the whole project, may cause confusion in the long term. 
But maybe this is a discussion for later.

Reply:
yea, i should remove it. good point